homeassistant/components/webostv/config_flow.py

```diff
@@ -188,7 +186,11 @@ async def async_step_init(
         if not sources_list:
             errors["base"] = "cannot_retrieve"
 
-        sources = [s for s in self.options.get(CONF_SOURCES, []) if s in sources_list]
+        sources = [
+            s
+            for s in self.config_entry.options.get(CONF_SOURCES, [])
+            if s in sources_list
+        ]
```

Comment:
As far as I know the guidelines in HA are to avoid using multi line ternary, so this can be changed to:
```suggestion
        options = self.config_entry.options.get(CONF_SOURCES, [])
        sources = [s for s in options if s in sources_list]
```